src/shim.php

```diff
@@ -1,40 +0,0 @@
-<?php
-if (!class_exists('PHPUnit\Framework\TestCase') && class_exists('PHPUnit_Framework_TestCase')) {
-    class_alias('PHPUnit_Framework_TestCase', 'PHPUnit'.'\Framework\TestCase');
-}
-
-if (!class_exists('PHPUnit\Runner\Version')) {
-    class_alias('PHPUnit_Runner_Version', 'PHPUnit\Runner\Version');
-}
-if (class_exists('PHPUnit_Framework_MockObject_Generator')) {
```

Comment:
After codeception and phpunit upgrade I started receiving an error while using php-vcr.
PHPUnit_Framework_MockObject_Generator class does not exist for me but it was still trying to autoload it and raised a runtime error:
[PHPUnit\Framework\Exception]
include(PHPUnit_Framework_MockObject_Generator.php): failed to open stream:
"VCR\Util\StreamProcessor::stream_open" call failed

stream_open function was trying to find PHPUnit_Framework_MockObject_Generator.php and returned false.

I am not sure why this error occurred only when running codeception tests with coverage